### PR TITLE
Updated  README.md

### LINKED ISSUE #129: Update README
#### Problem description
The project has been renamed to `skunkworks-crow` since it is still unreleased and an experimental project. README also needs to be updated. `Share` should be replaced with `skunkworks-crow`

### PR DESCRIPTION
**Pull request by iamvasuh**
Closes #129 

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by iamvasuh** [comment]
@shobhitagarwal1612  now u can merge this PR

**Comment by iamvasuh** [comment]
@shobhitagarwal1612  now u can merge this PR

File: README.md [hunk 1]
```diff
@@ -114,21 +114,21 @@ Is user wants to manually connect to it then the screen shows the network name s
...
 
 A receiver will need to connect to the network the sender has created. There are two ways to connect to the network:
 A QR code generated by the sender can be scanned by a receiver, it will get all the information required to connect to the sender (wifi name, port,  and password if the connection is protected).
 Tap on the wifi network available to connect and after successful connection enter port number to start receiving the forms.
 
 Connecting to the sender’s network will disable all the wifi network available so that no other network can interrupt the current connection. After receiving the forms it will restore the last saved state.
 
-For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-SHARE-odk-share” is set as the default network name and for versions >= 8 it is AndroidShare-XXXX which is set by Android.
+For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-SHARE-crow-odk-share” is set as the default network name and for versions >= 8 it is AndroidShare-XXXX which is set by Android.
```
Comment (shobhitagarwal1612) [inline README.md]: Please undo this as well. This PR does not modify the name of the wifi hotspot

Comment (shobhitagarwal1612) [reply README.md]: Please do not open a new pull request for making this change. You should push to the branch

**Comment by iamvasuh** [comment]
ok done now @shobhitagarwal1612

**Comment by shobhitagarwal1612** [comment]
Thanks @iamvasuh

**Pull request merged**